package/nginx-https.conf

```diff
@@ -12,6 +12,7 @@ server {
 
     ssl_certificate     /usr/share/rmt/ssl/rmt-server.crt;
     ssl_certificate_key /usr/share/rmt/ssl/rmt-server.key;
+    ssl_protocols       TLSv1.2 TLSv1.3;
```

Comment:
@kalabiyau I've thrown in TLSv1.3 as well, just for the future. Should the cipher suite also be modified?

Reply:
@ikapelyukhin 

I think default still makes sense and there was no specific advise from security team on that bit. So, I suggest we keep it this way for now.

```bash
# ssl_ciphers  HIGH:!aNULL:!MD5;
```

Reply:
OK! :+1: